src/Server/TCPServer.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <Poco/Net/TCPServer.h>
+
+#include <base/types.h>
+#include <Server/TCPServerConnectionFactory.h>
+
+
+namespace DB
+{
+class Context;
+
+class TCPServer : public Poco::Net::TCPServer
+{
+public:
+    explicit TCPServer(
+        TCPServerConnectionFactory::Ptr factory,
+        Poco::ThreadPool & thread_pool,
+        Poco::Net::ServerSocket & socket,
+        Poco::Net::TCPServerParams::Ptr params = new Poco::Net::TCPServerParams);
+
+    /// Close the socket and ask existing connections to stop serving queries
+    void close()
```

Comment:
`Poco::Net::TCPServer` already has the function `stop()`. It looks misleading to have both `stop()` and `close()` functions in `TCPServer`. Can we rename `close()` -> `stop()` and also call `Poco::Net::TCPServer::stop()` in this function?